src/particles/ImpactXParticleContainer.H

```diff
@@ -155,6 +160,17 @@ namespace impactx
                 amrex::ParticleReal, amrex::ParticleReal>
         MeanAndStdPositions ();
 
+        /** Deposit the charge of the particles onto a grid
+         *
+         * This resets the values in rho to zero and then deposits the particle
+         * charge.
+         *
+         * @param rho charge grid to deposit on
+         * @param icomp component in rho to store into
+         */
+        void
+        DepositCharge (amrex::MultiFab & rho, int icomp = 0);
```

Comment:
I think that we can get rid of `icomp` here. In the context of ImpactZ, this will always be `0`, so no need to expose it as an option.